src/discussions/comments/data/slices.js

```diff
@@ -115,14 +132,14 @@ const commentsSlice = createSlice({
 });
 
 export const {
-  fetchCommentDenied,
-  fetchCommentFailed,
-  fetchCommentRequest,
+  fetchCommentResponsesDenied,
+  fetchCommentResponsesFailed,
+  fetchCommentResponsesRequest,
+  fetchCommentResponsesSuccess,
   fetchCommentsDenied,
   fetchCommentsFailed,
   fetchCommentsRequest,
   fetchCommentsSuccess,
-  fetchCommentSuccess,
   postCommentDenied,
   postCommentFailed,
   postCommentRequest,
```

Comment:
The rest are mostly code changes due to renaming fetchComment* to the more accurate fetchCommentaResponses*